lib/models/rabbitmq.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "RabbitMQ",
+  "plural": "RabbitMQ",
+  "base": "Model",
+  "description": "Provide access to data about the queues and the rabbitmq server."
```

Comment:
Let's change the description: `Provide access to the queues and the RabbitMQ server.`. It says the same and make it fit on 1 line in the explorer.
<img width="967" alt="screen shot 2017-01-16 at 13 32 22" src="https://cloud.githubusercontent.com/assets/36491/21994950/4af59ea8-dbf0-11e6-8a3d-84686f3cc391.png">